bash_completion.sh

```diff
@@ -278,7 +278,7 @@ _buildtest ()
           esac
         ;;
         profiles)
-          local opts="--help -h list"
```

Comment:
please sort this alphabetically. So move `list` before `remove`. Keep the options as is. They need to be declared first